inlang/source-code/sdk/src/test-utilities/createMessage.test.ts

```diff
@@ -35,32 +52,31 @@ test("should create a message with pattern", () => {
 				{ type: "Text", value: '"' },
 			],
 		})
-	).toMatchInlineSnapshot(`
+	).toStrictEqual({
```

Comment:
I wasn't sure why we were string matching here but it seemed a bit off - do you know @martin-lysk ?

Reply:
TBH - no i think toMatchInlineSnapshot should be enough